_posts/2016-01-20-Making-of-Peacock.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Making of Peacock
+excerpt: Behind the scenes of making Swagger support in Apiary.
+layout: post
+date: 2016-01-20 00:00:00 +0100
+author: zdenek
+published: true
+---
+
+Recently, we have launched the biggest feature in the history of Apiary: [Swagger Support](http://blog.apiary.io/2016/01/18/We-ve-got-Swagger). This project was nicknamed "The Project Peacock".
+
+It took us almost three months to implement it and prepare for the launch.
```

Comment:
So, I wonder here if you could hint that we have been laying the groundwork for Peacock for a long time.